lib/einhorn/command.rb

```diff
@@ -349,6 +349,7 @@ def self.prepare_child_environment(index)
 
       ENV["EINHORN_FD_COUNT"] = Einhorn::State.bind_fds.length.to_s
       Einhorn::State.bind_fds.each_with_index { |fd, i| ENV["EINHORN_FD_#{i}"] = fd.to_s }
+      Einhorn::State.bind.each_with_index { |bind, i| ENV["EINHORN_FD_FAMILY_#{i}"] = bind.family.to_s }
```

Comment:
Added `to_s` here to match the line above, and because of the `Socket::AF_INET` thing above.

It is currently silly, and boils down to: `"AF_UNIX".to_s`